### ISSUE TITLE
[Footer]: QA: The mentioned page elements visually look like heading but screen reader does not identify these as heading

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8870]
### Summary

Page Element:
Select geographic area in the first modal and the selected region that appears in the second one.

Issue:
The mentioned texts visually look like heading but screen reader does not identify these as heading

Expected:
If these are supposed to be headings, SR should identify these as H2

User Impact:
When structural markups like headings and lists are not used, Screen readers cannot convey the semantic information of those elements. This impacts users who are blind.

Component(s) impacted: Footer (Default)

Browser: Firefox

Carbon for IBM.com version: v1.34.0-rc.0

Application/website: Carbon for IBM.com

Package: @carbon/ibmdotcom-web-components

CodeSandbox example: https://www.ibm.com/standards/carbon/web-components/?path=/story/

### Name and organization

Name: Gopinathan
Organization: IBM

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
